syncd/Syncd.cpp

```diff
@@ -3273,7 +3290,6 @@ sai_status_t Syncd::processNotifySyncd(
     {
         SWSS_LOG_NOTICE("very first run is TRUE, op = %s", key.c_str());
 
-        sai_status_t status = SAI_STATUS_SUCCESS;
```

Comment:
Please revert back the changes not relevant to this PR

Reply:
This change is needed because status is being defined above that would lead to compilation issue